apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -179,10 +179,6 @@ int tc_kernel_main(int argc, char *argv[])
 	timer_tc_main();
 #endif
 
-#ifdef CONFIG_TC_KERNEL_ROUNDROBIN
```

Comment:
1. Please check this config in Kconfig.
2. There is a typo in commit description, PAI -> API.

Reply:
modified